docs/components/AnnouncementBubbleView.jsx

```diff
@@ -125,6 +125,9 @@ export default class AnnouncementBubbleView extends React.PureComponent {
                 />
               }
               numTranslatedLanguages={1}
+              onTranslatedLanguagesClick={() =>
+                console.log("You clicked the language tooltip! Pitbull says dale!")
+              }
```

Comment:
I'm glad we're using the `onTranslatedLanguagesClick` prop in the examples! Shouldn't it also go in the list of props tho?